io/io/src/TDirectoryFile.cxx

```diff
@@ -740,12 +745,17 @@ void TDirectoryFile::FillBuffer(char *&buffer)
        fSeekKeys > TFile::kStartBigFile )
    {
       // One of the address is larger than 2GB we need to use longer onfile
-      // integer, thus we increase the verison number.
+      // integer, thus we increase the version number.
       // Note that fSeekDir and fSeekKey are not necessarily correlated, if
       // some object are 'removed' from the file and the holes are reused.
       version += 1000;
    }
    tobuf(buffer, version);
+   if (TestBit(TFile::kReproducible) || (fFile && fFile->TestBit(TFile::kReproducible))) {
+      fDatimeC = (UInt_t) 1;
+      fDatimeM = (UInt_t) 1;
+      fUUID.SetUUID("00000000-0000-0000-0000-000000000000");
```

Comment:
It should be documentation that because of this, support for TRef stored in a file 'reproducible' file will be at least degrated and possibly removed.

Reply:
Also see https://github.com/root-project/root/pull/4253/files#r316332485